packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -190,7 +203,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
     protected readonly summarizerNode: ISummarizerNodeWithGC;
 
     constructor(
-        private readonly _containerRuntime: ContainerRuntime,
+        protected readonly _containerRuntime: ContainerRuntime,
```

Comment:
It may be worth adding a protected getter `disableIsolatedChannels` to this base class that returns _containerRuntime.disableIsolatedChannels instead of making this protected.

Reply:
Good point.